packages/ag-charts-community/src/chart/agChartV2.ts

```diff
@@ -1,3 +1,4 @@
+import type { ModuleContext } from '../module-support';
```

Comment:
Nit: Could you use a more-specific import, rather than this large bundle? (e.g. from `./module/moduleContext` directly)?

(inside the same conceptual build unit, it's better to avoid bundles from a potential circular dependency perspective).